src/renderers/testing/__tests__/ReactTestRenderer-test.js

```diff
@@ -231,6 +232,84 @@ describe('ReactTestRenderer', () => {
     );
   });
 
+  it('allows an optional getMockRef function', () => {
```

Comment:
Currently if you pass in anything as options to `create` it does assume that you've provided a `getMockRef` function and it will throw. How do you want to handle that? Should `getMockRef` be required if you're passing in your own options? 

I'm assuming somewhere down the line we provide more user-configurable options to pass here, so I don't want to default to `defaultMockConfig` if `getMockRef` doesn't exist.